### PR TITLE
Automate Underwater Marauder

### PR DESCRIPTION
**Pull request by TikaelSol**

### CODE REVIEW

**Comment by simonward** [comment]
Consider localizing the Note:
"You can't cast fire spells or use actions with the fire trait underwater."

**Comment by TikaelSol** [comment]
Done

**Review by simonward — APPROVED** [review]

File: packs/data/feats.db/underwater-marauder.json [hunk 1]
```diff
@@ -23,7 +23,23 @@
...
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "option": "aquatic-combat:no-penalty-bludgeoning"
+            },
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "option": "aquatic-combat:no-penalty-slashing"
+            }
```
Comment (stwlam) [inline packs/data/feats.db/underwater-marauder.json:41]: These two would be better handled with modifier suppression. The Aquatic Combat effect wouldn't need its flat modifier REs altered except to add slugs.

Comment (TikaelSol) [reply packs/data/feats.db/underwater-marauder.json:41]: Good point, was trying to prep a session at the last minute and completely forgot we could suppress with AdjustModifier. I'll rework with that.

**Comment by simonward** [comment]
Do the selectors need to match for Underwater Marauder and Effect: Underwater Combat?
`"selector": "strike-attack-roll"`
`"selector": "attack"`

File: packs/data/feats.db/underwater-marauder.json [hunk 2]
```diff
@@ -23,7 +23,21 @@
...
+        "rules": [
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "option": "aquatic-combat:not-flat-footed"
+            },
+            {
+                "key": "AdjustModifier",
+                "mode": "override",
+                "selector": "strike-attack-roll",
```
Comment (TikaelSol) [inline packs/data/feats.db/underwater-marauder.json:35]:
```suggestion
                "selector": "attack",
```

File: packs/data/feats.db/underwater-marauder.json [hunk 3]
```diff
@@ -23,7 +23,21 @@
...
+                "key": "RollOption",
+                "option": "aquatic-combat:not-flat-footed"
+            },
+            {
+                "key": "AdjustModifier",
+                "mode": "override",
+                "selector": "attack",
+                "slug": "aquatic-combat",
+                "suppress": true,
+                "value": 0
```
Comment (stwlam) [inline packs/data/feats.db/underwater-marauder.json:38]: Was adding a value necessary for it to work? That's a bug in the RE code if so

Comment (TikaelSol) [inline packs/data/feats.db/underwater-marauder.json:38]:
```suggestion
                "suppress": true
```

**Comment by TikaelSol** [comment]
Done

**Review by simonward — APPROVED** [review]

**Pull request merged**